ray-operator/test/e2erayservice/rayservice_upgrade_test.go

```diff
@@ -130,7 +130,7 @@ func TestOldHeadPodFailDuringUpgrade(t *testing.T) {
 		g.Expect(endpoints.Subsets).To(HaveLen(1))
 		g.Expect(endpoints.Subsets[0].Addresses).To(HaveLen(1))
 		g.Expect(endpoints.Subsets[0].Addresses[0].TargetRef.Name).NotTo(Equal(headPodName))
-	}, TestTimeoutShort).Should(Succeed())
+	}, TestTimeoutMedium).Should(Succeed())
```

Comment:
On my devbox, this step typically takes 50 to 55 seconds to finish, and the `TestTimeoutShort` is 60 seconds by default. It timed out during the last CI run. Increase it to 2 minutes.

Reply:
FYI @rueian